docs/products/opensearch/dashboards/index.rst

```diff
@@ -0,0 +1,31 @@
+OpenSearch Dashboards
+======================
+
+OpenSearch Dashboards is both a visualisation tool for data in the cluster and a user interface for OpenSearch plugins.
+
+.. note::
+    OpenSearch Dashboards was forked in 2021 from the formerly open source project Kibana.
+
+Get started with Aiven for OpenSearch Dashboards
+------------------------------------------------
+
+Take your first steps with Aiven for OpenSearch Dashboards by following our :doc:`getting-started` article, or browse through the full list of articles:
+
+.. panels::
```

Comment:
I would skip this until we've got a bit more to put here. Maybe a link to the HowTos in the bottom section with the other links for now?